package.json

```diff
@@ -21,6 +21,9 @@
       "url": "https://raw.githubusercontent.com/gilbitron/Raneto/master/LICENSE"
     }
   ],
+  "bin": {
+    "raneto": "./bin/raneto.js"
+  },
```

Comment:
I wouldn't deny a request for this one thing, but while you're editing anyway, change this to:
"bin": "./bin/raneto" as per NPM spec (defaults to name of package)